packages/fxa-content-server/app/scripts/views/settings.js

```diff
@@ -78,6 +79,11 @@ const View = BaseView.extend({
     this._subscriptionsManagementLanguages =
       options.subscriptionsManagementLanguages;
 
+    if (options.config && options.config.subscriptions) {
+      this._subscriptionManagementUrl =
+        options.config.subscriptions.managementUrl;
```

Comment:
I just read about #1850 in my bugmail, so it seems likely this doesn't do what I think it does.

@lmorchard, do you know what this URL is for, if it's not for #1850?

Reply:
Oh yeah, sorry - missed this comment until now :)